lib/App/index.js

```diff
@@ -56,16 +58,72 @@ const FLOW_TYPES = [ 'triggers', 'conditions', 'actions' ];
 class App {
 
 	constructor( appPath ) {
-		this.path = path.resolve(appPath);
-		this._app = new HomeyLibApp( this.path );
+    this.path = path.resolve(appPath);
+    this.srcPath = this.path;
+    this._app = new HomeyLibApp( this.path );
     this._appJsonPath = path.join( this.path, 'app.json' );
     this._pluginsPath = path.join( this.path, '.homeyplugins.json');
     this._homeyComposePath = path.join(this.path, '.homeycompose');
-		this._exiting = false;
-		this._std = {};
+    this._exiting = false;
+    this._std = {};
     this._git = new GitCommands(appPath);
   }
 
+
+  /* 
+  * Returns true if typescript is set to true in the app.json 
+  */
+  async usesTypeScript() {
+    const json = await this.getAppJson();
+    return json.typescript;
+  }
```

Comment:
We should probably check `package.json` for an installed `typescript`?
Maybe we should even implement this as build step so people can use babel etc with the homey-cli?
Something like the npm `prepare` script in package.json?

Reply:
I am not sure I like the addition of the `typescript` property to `app.json`. Can't we better check for the existence of a `.tsconfig` file?